INFO 2025-09-24 07:23:34,575 autoreload 23616 29536 Watching for file changes with StatReloader
INFO 2025-09-24 07:25:06,451 autoreload 17760 26828 Watching for file changes with StatReloader
INFO 2025-09-24 07:29:31,018 autoreload 17760 26828 C:\Users\Perdoruess\bottlebrothers_lab-7\server\api\views.py changed, reloading.
INFO 2025-09-24 07:29:33,942 autoreload 23780 19204 Watching for file changes with StatReloader
INFO 2025-09-24 07:32:01,650 autoreload 23780 19204 C:\Users\Perdoruess\bottlebrothers_lab-7\server\api\serializers.py changed, reloading.
INFO 2025-09-24 07:32:05,473 autoreload 13512 21496 Watching for file changes with StatReloader
INFO 2025-09-24 07:32:30,059 autoreload 13512 21496 C:\Users\Perdoruess\bottlebrothers_lab-7\server\api\views.py changed, reloading.
INFO 2025-09-24 07:32:34,428 autoreload 10944 24824 Watching for file changes with StatReloader
INFO 2025-09-24 07:33:32,201 autoreload 10944 24824 C:\Users\Perdoruess\bottlebrothers_lab-7\server\api\admin.py changed, reloading.
INFO 2025-09-24 07:33:34,847 autoreload 17712 23972 Watching for file changes with StatReloader
INFO 2025-09-24 07:34:03,889 autoreload 17712 23972 C:\Users\Perdoruess\bottlebrothers_lab-7\server\api\models.py changed, reloading.
INFO 2025-09-24 07:34:05,469 autoreload 6296 30156 Watching for file changes with StatReloader
INFO 2025-09-24 07:37:45,262 autoreload 6296 30156 C:\Users\Perdoruess\bottlebrothers_lab-7\server\server\settings.py changed, reloading.
INFO 2025-09-24 07:39:20,263 autoreload 22020 26224 Watching for file changes with StatReloader
INFO 2025-09-24 07:39:48,071 autoreload 21612 28908 Watching for file changes with StatReloader
INFO 2025-09-24 07:40:38,355 autoreload 21612 28908 C:\Users\Perdoruess\bottlebrothers_lab-7\server\api\admin.py changed, reloading.
INFO 2025-09-24 07:40:39,573 autoreload 3020 25352 Watching for file changes with StatReloader
INFO 2025-09-24 07:41:22,083 autoreload 9332 29684 Watching for file changes with StatReloader
INFO 2025-09-24 07:57:48,832 basehttp 9332 29692 "OPTIONS /api/reservations/dashboard/ HTTP/1.1" 200 0
WARNING 2025-09-24 07:57:49,238 log 9332 29796 Unauthorized: /api/reservations/dashboard/
WARNING 2025-09-24 07:57:49,243 basehttp 9332 29796 "GET /api/reservations/dashboard/ HTTP/1.1" 401 183
WARNING 2025-09-24 07:57:49,249 log 9332 29796 Unauthorized: /api/reservations/dashboard/
INFO 2025-09-24 07:57:49,252 basehttp 9332 29796 - Broken pipe from ('127.0.0.1', 51269)
INFO 2025-09-24 07:58:00,120 basehttp 9332 29692 "OPTIONS /api/auth/login/ HTTP/1.1" 200 0
INFO 2025-09-24 07:58:04,827 basehttp 9332 29692 "POST /api/auth/login/ HTTP/1.1" 200 626
INFO 2025-09-24 07:58:04,963 basehttp 9332 29692 "OPTIONS /api/reservations/dashboard/ HTTP/1.1" 200 0
WARNING 2025-09-24 07:58:04,974 log 9332 29692 Not Found: /api/reservations/dashboard/
INFO 2025-09-24 07:58:04,975 basehttp 9332 20452 "OPTIONS /api/reservations/dashboard/ HTTP/1.1" 200 0
WARNING 2025-09-24 07:58:04,980 basehttp 9332 29692 "GET /api/reservations/dashboard/ HTTP/1.1" 404 23
WARNING 2025-09-24 07:58:04,988 log 9332 20452 Not Found: /api/reservations/dashboard/
WARNING 2025-09-24 07:58:04,997 basehttp 9332 20452 "GET /api/reservations/dashboard/ HTTP/1.1" 404 23
INFO 2025-09-24 07:58:19,001 basehttp 9332 29692 "OPTIONS /api/reservations/ HTTP/1.1" 200 0
INFO 2025-09-24 07:58:19,023 basehttp 9332 29692 "GET /api/reservations/ HTTP/1.1" 200 52
INFO 2025-09-24 07:58:19,043 basehttp 9332 29692 "GET /api/reservations/ HTTP/1.1" 200 52
WARNING 2025-09-24 07:59:42,711 log 9332 29692 Not Found: /api/reservations/dashboard/
WARNING 2025-09-24 07:59:42,713 log 9332 20452 Not Found: /api/reservations/dashboard/
WARNING 2025-09-24 07:59:42,718 basehttp 9332 29692 "GET /api/reservations/dashboard/ HTTP/1.1" 404 23
WARNING 2025-09-24 07:59:42,725 basehttp 9332 20452 "GET /api/reservations/dashboard/ HTTP/1.1" 404 23
INFO 2025-09-24 07:59:46,021 basehttp 9332 29692 "GET /api/reservations/ HTTP/1.1" 200 52
INFO 2025-09-24 07:59:46,034 basehttp 9332 29692 "GET /api/reservations/ HTTP/1.1" 200 52
INFO 2025-09-24 08:08:26,334 autoreload 9332 29684 C:\Users\Perdoruess\bottlebrothers_lab-7\server\api\views.py changed, reloading.
INFO 2025-09-24 08:08:30,206 autoreload 14564 20676 Watching for file changes with StatReloader
INFO 2025-09-24 08:08:39,704 autoreload 14564 20676 C:\Users\Perdoruess\bottlebrothers_lab-7\server\api\urls.py changed, reloading.
INFO 2025-09-24 08:08:42,515 autoreload 3048 22992 Watching for file changes with StatReloader
INFO 2025-09-24 08:31:48,076 basehttp 3048 19308 "GET /api/reservations/ HTTP/1.1" 200 52
INFO 2025-09-24 08:31:48,105 basehttp 3048 19308 "GET /api/reservations/ HTTP/1.1" 200 52
INFO 2025-09-26 07:58:20,955 autoreload 3048 22992 C:\Users\Perdoruess\bottlebrothers_lab-7\server\api\serializers.py changed, reloading.
INFO 2025-09-26 07:58:32,737 autoreload 27460 30264 Watching for file changes with StatReloader
INFO 2025-09-26 10:36:34,448 autoreload 19640 20932 Watching for file changes with StatReloader
INFO 2025-09-26 10:36:37,046 basehttp 19640 10560 "GET / HTTP/1.1" 200 282
WARNING 2025-09-26 10:36:37,569 log 19640 14676 Not Found: /favicon.ico
WARNING 2025-09-26 10:36:37,575 basehttp 19640 14676 "GET /favicon.ico HTTP/1.1" 404 2440
INFO 2025-09-26 10:36:51,234 basehttp 19640 6160 "OPTIONS /api/reservations/dashboard/ HTTP/1.1" 200 0
INFO 2025-09-26 10:36:51,234 basehttp 19640 20316 "OPTIONS /api/reservations/dashboard/ HTTP/1.1" 200 0
WARNING 2025-09-26 10:36:51,688 log 19640 6160 Unauthorized: /api/reservations/dashboard/
WARNING 2025-09-26 10:36:51,691 log 19640 20316 Unauthorized: /api/reservations/dashboard/
WARNING 2025-09-26 10:36:51,693 basehttp 19640 6160 "GET /api/reservations/dashboard/ HTTP/1.1" 401 183
INFO 2025-09-26 10:36:51,714 basehttp 19640 20316 - Broken pipe from ('127.0.0.1', 50589)
INFO 2025-09-26 12:25:05,478 basehttp 19640 26736 "OPTIONS /api/events/public/ HTTP/1.1" 200 0
INFO 2025-09-26 12:25:05,496 basehttp 19640 24972 "GET /api/events/public/ HTTP/1.1" 200 2
INFO 2025-09-26 12:25:05,529 basehttp 19640 24972 "GET /api/events/public/ HTTP/1.1" 200 2
INFO 2025-09-26 12:25:09,462 basehttp 19640 24972 "OPTIONS /api/gallery/public/ HTTP/1.1" 200 0
INFO 2025-09-26 12:25:09,482 basehttp 19640 24972 "GET /api/gallery/public/ HTTP/1.1" 200 458
INFO 2025-09-26 12:25:09,498 basehttp 19640 24972 "GET /api/gallery/public/ HTTP/1.1" 200 458
WARNING 2025-09-26 12:25:09,850 log 19640 29068 Not Found: /media/gallery/4977b4d89700444f99604571036d7f33.png
WARNING 2025-09-26 12:25:09,851 basehttp 19640 29068 "GET /media/gallery/4977b4d89700444f99604571036d7f33.png HTTP/1.1" 404 2805
INFO 2025-09-26 12:25:10,133 basehttp 19640 10552 "GET /media/gallery/ca9f94b1f91b4871b86764436f871e5d.jpg HTTP/1.1" 200 22272
INFO 2025-09-26 12:25:15,613 basehttp 19640 24972 "GET /api/events/public/ HTTP/1.1" 200 2
INFO 2025-09-26 12:25:15,628 basehttp 19640 24972 "GET /api/events/public/ HTTP/1.1" 200 2
INFO 2025-09-26 12:28:05,974 autoreload 15812 22264 Watching for file changes with StatReloader
INFO 2025-09-26 12:28:44,908 basehttp 15812 6180 "GET /api/events/public/ HTTP/1.1" 200 2
INFO 2025-09-26 12:28:44,937 basehttp 15812 6180 "GET /api/events/public/ HTTP/1.1" 200 2
INFO 2025-09-26 12:46:15,928 basehttp 15812 6092 "GET /api/events/public/ HTTP/1.1" 200 2
INFO 2025-09-26 12:46:15,968 basehttp 15812 6092 "GET /api/events/public/ HTTP/1.1" 200 2
INFO 2025-09-26 12:48:35,987 basehttp 15812 14640 "GET /api/events/public/ HTTP/1.1" 200 2
INFO 2025-09-26 12:48:36,009 basehttp 15812 14640 "GET /api/events/public/ HTTP/1.1" 200 2
INFO 2025-09-26 12:52:00,020 basehttp 15812 332 "GET /api/events/public/ HTTP/1.1" 200 2
INFO 2025-09-26 12:52:00,052 basehttp 15812 332 "GET /api/events/public/ HTTP/1.1" 200 2
INFO 2025-09-26 12:52:21,753 basehttp 15812 332 "OPTIONS /api/auth/login/ HTTP/1.1" 200 0
WARNING 2025-09-26 12:52:22,912 log 15812 332 Bad Request: /api/auth/login/
WARNING 2025-09-26 12:52:22,913 basehttp 15812 332 "POST /api/auth/login/ HTTP/1.1" 400 44
INFO 2025-09-26 12:53:22,587 autoreload 20516 10452 Watching for file changes with StatReloader
INFO 2025-09-26 12:53:37,476 basehttp 20516 27592 "POST /api/auth/login/ HTTP/1.1" 200 640
INFO 2025-09-26 12:53:37,704 basehttp 20516 27592 "OPTIONS /api/reservations/dashboard/ HTTP/1.1" 200 0
INFO 2025-09-26 12:53:37,742 basehttp 20516 27592 "OPTIONS /api/reservations/dashboard/ HTTP/1.1" 200 0
WARNING 2025-09-26 12:53:37,745 log 20516 24264 Not Found: /api/reservations/dashboard/
WARNING 2025-09-26 12:53:37,770 log 20516 27592 Not Found: /api/reservations/dashboard/
WARNING 2025-09-26 12:53:37,774 basehttp 20516 24264 "GET /api/reservations/dashboard/ HTTP/1.1" 404 23
WARNING 2025-09-26 12:53:37,802 basehttp 20516 27592 "GET /api/reservations/dashboard/ HTTP/1.1" 404 23
INFO 2025-09-26 12:53:40,324 basehttp 20516 24264 "OPTIONS /api/contact/ HTTP/1.1" 200 0
INFO 2025-09-26 12:53:40,343 basehttp 20516 24264 "GET /api/contact/ HTTP/1.1" 200 52
INFO 2025-09-26 12:53:40,376 basehttp 20516 24264 "GET /api/contact/ HTTP/1.1" 200 52
INFO 2025-09-26 12:53:42,841 basehttp 20516 24264 "OPTIONS /api/events/ HTTP/1.1" 200 0
INFO 2025-09-26 12:53:42,862 basehttp 20516 24264 "GET /api/events/ HTTP/1.1" 200 2
INFO 2025-09-26 12:53:42,890 basehttp 20516 24264 "GET /api/events/ HTTP/1.1" 200 2
WARNING 2025-09-26 12:55:37,518 log 20516 24264 Bad Request: /api/events/
WARNING 2025-09-26 12:55:37,519 basehttp 20516 24264 "POST /api/events/ HTTP/1.1" 400 105
WARNING 2025-09-26 13:26:45,839 log 20516 16560 Bad Request: /api/events/
WARNING 2025-09-26 13:26:45,840 basehttp 20516 16560 "POST /api/events/ HTTP/1.1" 400 105
INFO 2025-09-26 13:28:16,387 basehttp 20516 16560 "GET /api/events/ HTTP/1.1" 200 2
INFO 2025-09-26 13:28:16,406 basehttp 20516 23160 "GET /api/events/ HTTP/1.1" 200 2
INFO 2025-09-26 13:28:17,904 basehttp 20516 23160 "GET /api/events/public/ HTTP/1.1" 200 2
INFO 2025-09-26 13:28:17,914 basehttp 20516 23160 "GET /api/events/public/ HTTP/1.1" 200 2
ERROR 2025-09-26 13:28:59,504 log 20516 16560 Internal Server Error: /api/events/
Traceback (most recent call last):
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\django\views\decorators\csrf.py", line 65, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\viewsets.py", line 125, in view
    return self.dispatch(request, *args, **kwargs)
           ~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\views.py", line 480, in raise_uncaught_exception
    raise exc
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\mixins.py", line 20, in create
    headers = self.get_success_headers(serializer.data)
                                       ^^^^^^^^^^^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\serializers.py", line 555, in data
    ret = super().data
          ^^^^^^^^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\serializers.py", line 253, in data
    self._data = self.to_representation(self.instance)
                 ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\serializers.py", line 509, in to_representation
    attribute = field.get_attribute(instance)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\fields.py", line 446, in get_attribute
    return get_attribute(instance, self.source_attrs)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\fields.py", line 96, in get_attribute
    instance = getattr(instance, attr)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\api\models.py", line 388, in formatted_time
    start_str = self.start_time.strftime('%-I:%M %p')
ValueError: Invalid format string
ERROR 2025-09-26 13:28:59,555 basehttp 20516 16560 "POST /api/events/ HTTP/1.1" 500 149547
ERROR 2025-09-26 13:28:59,653 log 20516 23160 Internal Server Error: /api/events/
Traceback (most recent call last):
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\django\views\decorators\csrf.py", line 65, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\viewsets.py", line 125, in view
    return self.dispatch(request, *args, **kwargs)
           ~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\views.py", line 480, in raise_uncaught_exception
    raise exc
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\mixins.py", line 20, in create
    headers = self.get_success_headers(serializer.data)
                                       ^^^^^^^^^^^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\serializers.py", line 555, in data
    ret = super().data
          ^^^^^^^^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\serializers.py", line 253, in data
    self._data = self.to_representation(self.instance)
                 ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\serializers.py", line 509, in to_representation
    attribute = field.get_attribute(instance)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\fields.py", line 446, in get_attribute
    return get_attribute(instance, self.source_attrs)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\fields.py", line 96, in get_attribute
    instance = getattr(instance, attr)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\api\models.py", line 388, in formatted_time
    start_str = self.start_time.strftime('%-I:%M %p')
ValueError: Invalid format string
ERROR 2025-09-26 13:28:59,707 basehttp 20516 23160 "POST /api/events/ HTTP/1.1" 500 149547
ERROR 2025-09-26 13:29:00,007 log 20516 16560 Internal Server Error: /api/events/
Traceback (most recent call last):
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\django\views\decorators\csrf.py", line 65, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\viewsets.py", line 125, in view
    return self.dispatch(request, *args, **kwargs)
           ~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\views.py", line 480, in raise_uncaught_exception
    raise exc
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\mixins.py", line 20, in create
    headers = self.get_success_headers(serializer.data)
                                       ^^^^^^^^^^^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\serializers.py", line 555, in data
    ret = super().data
          ^^^^^^^^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\serializers.py", line 253, in data
    self._data = self.to_representation(self.instance)
                 ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\serializers.py", line 509, in to_representation
    attribute = field.get_attribute(instance)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\fields.py", line 446, in get_attribute
    return get_attribute(instance, self.source_attrs)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\fields.py", line 96, in get_attribute
    instance = getattr(instance, attr)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\api\models.py", line 388, in formatted_time
    start_str = self.start_time.strftime('%-I:%M %p')
ValueError: Invalid format string
ERROR 2025-09-26 13:29:00,036 basehttp 20516 16560 "POST /api/events/ HTTP/1.1" 500 149547
ERROR 2025-09-26 13:29:00,886 log 20516 23160 Internal Server Error: /api/events/
Traceback (most recent call last):
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\django\views\decorators\csrf.py", line 65, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\viewsets.py", line 125, in view
    return self.dispatch(request, *args, **kwargs)
           ~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\views.py", line 480, in raise_uncaught_exception
    raise exc
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\mixins.py", line 20, in create
    headers = self.get_success_headers(serializer.data)
                                       ^^^^^^^^^^^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\serializers.py", line 555, in data
    ret = super().data
          ^^^^^^^^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\serializers.py", line 253, in data
    self._data = self.to_representation(self.instance)
                 ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\serializers.py", line 509, in to_representation
    attribute = field.get_attribute(instance)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\fields.py", line 446, in get_attribute
    return get_attribute(instance, self.source_attrs)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\fields.py", line 96, in get_attribute
    instance = getattr(instance, attr)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\api\models.py", line 388, in formatted_time
    start_str = self.start_time.strftime('%-I:%M %p')
ValueError: Invalid format string
ERROR 2025-09-26 13:29:00,895 basehttp 20516 23160 "POST /api/events/ HTTP/1.1" 500 149546
ERROR 2025-09-26 13:29:01,296 log 20516 16560 Internal Server Error: /api/events/
Traceback (most recent call last):
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\django\views\decorators\csrf.py", line 65, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\viewsets.py", line 125, in view
    return self.dispatch(request, *args, **kwargs)
           ~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\views.py", line 480, in raise_uncaught_exception
    raise exc
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\mixins.py", line 20, in create
    headers = self.get_success_headers(serializer.data)
                                       ^^^^^^^^^^^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\serializers.py", line 555, in data
    ret = super().data
          ^^^^^^^^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\serializers.py", line 253, in data
    self._data = self.to_representation(self.instance)
                 ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\serializers.py", line 509, in to_representation
    attribute = field.get_attribute(instance)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\fields.py", line 446, in get_attribute
    return get_attribute(instance, self.source_attrs)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\fields.py", line 96, in get_attribute
    instance = getattr(instance, attr)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\api\models.py", line 388, in formatted_time
    start_str = self.start_time.strftime('%-I:%M %p')
ValueError: Invalid format string
ERROR 2025-09-26 13:29:01,304 basehttp 20516 16560 "POST /api/events/ HTTP/1.1" 500 149547
ERROR 2025-09-26 13:29:01,472 log 20516 23160 Internal Server Error: /api/events/
Traceback (most recent call last):
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\django\views\decorators\csrf.py", line 65, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\viewsets.py", line 125, in view
    return self.dispatch(request, *args, **kwargs)
           ~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\views.py", line 480, in raise_uncaught_exception
    raise exc
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\mixins.py", line 20, in create
    headers = self.get_success_headers(serializer.data)
                                       ^^^^^^^^^^^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\serializers.py", line 555, in data
    ret = super().data
          ^^^^^^^^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\serializers.py", line 253, in data
    self._data = self.to_representation(self.instance)
                 ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\serializers.py", line 509, in to_representation
    attribute = field.get_attribute(instance)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\fields.py", line 446, in get_attribute
    return get_attribute(instance, self.source_attrs)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\fields.py", line 96, in get_attribute
    instance = getattr(instance, attr)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\api\models.py", line 388, in formatted_time
    start_str = self.start_time.strftime('%-I:%M %p')
ValueError: Invalid format string
ERROR 2025-09-26 13:29:01,497 basehttp 20516 23160 "POST /api/events/ HTTP/1.1" 500 149547
ERROR 2025-09-26 13:29:01,628 log 20516 16560 Internal Server Error: /api/events/
Traceback (most recent call last):
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\django\views\decorators\csrf.py", line 65, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\viewsets.py", line 125, in view
    return self.dispatch(request, *args, **kwargs)
           ~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\views.py", line 480, in raise_uncaught_exception
    raise exc
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\mixins.py", line 20, in create
    headers = self.get_success_headers(serializer.data)
                                       ^^^^^^^^^^^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\serializers.py", line 555, in data
    ret = super().data
          ^^^^^^^^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\serializers.py", line 253, in data
    self._data = self.to_representation(self.instance)
                 ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\serializers.py", line 509, in to_representation
    attribute = field.get_attribute(instance)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\fields.py", line 446, in get_attribute
    return get_attribute(instance, self.source_attrs)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\fields.py", line 96, in get_attribute
    instance = getattr(instance, attr)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\api\models.py", line 388, in formatted_time
    start_str = self.start_time.strftime('%-I:%M %p')
ValueError: Invalid format string
ERROR 2025-09-26 13:29:01,636 basehttp 20516 16560 "POST /api/events/ HTTP/1.1" 500 149547
ERROR 2025-09-26 13:29:01,770 log 20516 23160 Internal Server Error: /api/events/
Traceback (most recent call last):
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\django\views\decorators\csrf.py", line 65, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\viewsets.py", line 125, in view
    return self.dispatch(request, *args, **kwargs)
           ~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\views.py", line 480, in raise_uncaught_exception
    raise exc
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\mixins.py", line 20, in create
    headers = self.get_success_headers(serializer.data)
                                       ^^^^^^^^^^^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\serializers.py", line 555, in data
    ret = super().data
          ^^^^^^^^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\serializers.py", line 253, in data
    self._data = self.to_representation(self.instance)
                 ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\serializers.py", line 509, in to_representation
    attribute = field.get_attribute(instance)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\fields.py", line 446, in get_attribute
    return get_attribute(instance, self.source_attrs)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\fields.py", line 96, in get_attribute
    instance = getattr(instance, attr)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\api\models.py", line 388, in formatted_time
    start_str = self.start_time.strftime('%-I:%M %p')
ValueError: Invalid format string
ERROR 2025-09-26 13:29:01,780 basehttp 20516 23160 "POST /api/events/ HTTP/1.1" 500 149547
ERROR 2025-09-26 13:29:01,897 log 20516 16560 Internal Server Error: /api/events/
Traceback (most recent call last):
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\django\views\decorators\csrf.py", line 65, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\viewsets.py", line 125, in view
    return self.dispatch(request, *args, **kwargs)
           ~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\views.py", line 480, in raise_uncaught_exception
    raise exc
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\mixins.py", line 20, in create
    headers = self.get_success_headers(serializer.data)
                                       ^^^^^^^^^^^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\serializers.py", line 555, in data
    ret = super().data
          ^^^^^^^^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\serializers.py", line 253, in data
    self._data = self.to_representation(self.instance)
                 ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\serializers.py", line 509, in to_representation
    attribute = field.get_attribute(instance)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\fields.py", line 446, in get_attribute
    return get_attribute(instance, self.source_attrs)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\fields.py", line 96, in get_attribute
    instance = getattr(instance, attr)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\api\models.py", line 388, in formatted_time
    start_str = self.start_time.strftime('%-I:%M %p')
ValueError: Invalid format string
ERROR 2025-09-26 13:29:01,906 basehttp 20516 16560 "POST /api/events/ HTTP/1.1" 500 149547
ERROR 2025-09-26 13:29:02,046 log 20516 23160 Internal Server Error: /api/events/
Traceback (most recent call last):
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\django\views\decorators\csrf.py", line 65, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\viewsets.py", line 125, in view
    return self.dispatch(request, *args, **kwargs)
           ~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\views.py", line 480, in raise_uncaught_exception
    raise exc
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\mixins.py", line 20, in create
    headers = self.get_success_headers(serializer.data)
                                       ^^^^^^^^^^^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\serializers.py", line 555, in data
    ret = super().data
          ^^^^^^^^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\serializers.py", line 253, in data
    self._data = self.to_representation(self.instance)
                 ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\serializers.py", line 509, in to_representation
    attribute = field.get_attribute(instance)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\fields.py", line 446, in get_attribute
    return get_attribute(instance, self.source_attrs)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\fields.py", line 96, in get_attribute
    instance = getattr(instance, attr)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\api\models.py", line 388, in formatted_time
    start_str = self.start_time.strftime('%-I:%M %p')
ValueError: Invalid format string
ERROR 2025-09-26 13:29:02,090 basehttp 20516 23160 "POST /api/events/ HTTP/1.1" 500 149547
ERROR 2025-09-26 13:29:02,183 log 20516 16560 Internal Server Error: /api/events/
Traceback (most recent call last):
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\django\views\decorators\csrf.py", line 65, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\viewsets.py", line 125, in view
    return self.dispatch(request, *args, **kwargs)
           ~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\views.py", line 480, in raise_uncaught_exception
    raise exc
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\mixins.py", line 20, in create
    headers = self.get_success_headers(serializer.data)
                                       ^^^^^^^^^^^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\serializers.py", line 555, in data
    ret = super().data
          ^^^^^^^^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\serializers.py", line 253, in data
    self._data = self.to_representation(self.instance)
                 ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\serializers.py", line 509, in to_representation
    attribute = field.get_attribute(instance)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\fields.py", line 446, in get_attribute
    return get_attribute(instance, self.source_attrs)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\fields.py", line 96, in get_attribute
    instance = getattr(instance, attr)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\api\models.py", line 388, in formatted_time
    start_str = self.start_time.strftime('%-I:%M %p')
ValueError: Invalid format string
ERROR 2025-09-26 13:29:02,193 basehttp 20516 16560 "POST /api/events/ HTTP/1.1" 500 149547
ERROR 2025-09-26 13:29:02,336 log 20516 23160 Internal Server Error: /api/events/
Traceback (most recent call last):
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\django\views\decorators\csrf.py", line 65, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\viewsets.py", line 125, in view
    return self.dispatch(request, *args, **kwargs)
           ~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\views.py", line 480, in raise_uncaught_exception
    raise exc
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\mixins.py", line 20, in create
    headers = self.get_success_headers(serializer.data)
                                       ^^^^^^^^^^^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\serializers.py", line 555, in data
    ret = super().data
          ^^^^^^^^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\serializers.py", line 253, in data
    self._data = self.to_representation(self.instance)
                 ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\serializers.py", line 509, in to_representation
    attribute = field.get_attribute(instance)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\fields.py", line 446, in get_attribute
    return get_attribute(instance, self.source_attrs)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\fields.py", line 96, in get_attribute
    instance = getattr(instance, attr)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\api\models.py", line 388, in formatted_time
    start_str = self.start_time.strftime('%-I:%M %p')
ValueError: Invalid format string
ERROR 2025-09-26 13:29:02,391 basehttp 20516 23160 "POST /api/events/ HTTP/1.1" 500 149547
ERROR 2025-09-26 13:29:02,511 log 20516 16560 Internal Server Error: /api/events/
Traceback (most recent call last):
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\django\views\decorators\csrf.py", line 65, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\viewsets.py", line 125, in view
    return self.dispatch(request, *args, **kwargs)
           ~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\views.py", line 480, in raise_uncaught_exception
    raise exc
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\mixins.py", line 20, in create
    headers = self.get_success_headers(serializer.data)
                                       ^^^^^^^^^^^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\serializers.py", line 555, in data
    ret = super().data
          ^^^^^^^^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\serializers.py", line 253, in data
    self._data = self.to_representation(self.instance)
                 ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\serializers.py", line 509, in to_representation
    attribute = field.get_attribute(instance)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\fields.py", line 446, in get_attribute
    return get_attribute(instance, self.source_attrs)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\fields.py", line 96, in get_attribute
    instance = getattr(instance, attr)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\api\models.py", line 388, in formatted_time
    start_str = self.start_time.strftime('%-I:%M %p')
ValueError: Invalid format string
ERROR 2025-09-26 13:29:02,581 basehttp 20516 16560 "POST /api/events/ HTTP/1.1" 500 149547
ERROR 2025-09-26 13:29:02,672 log 20516 23160 Internal Server Error: /api/events/
Traceback (most recent call last):
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\django\views\decorators\csrf.py", line 65, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\viewsets.py", line 125, in view
    return self.dispatch(request, *args, **kwargs)
           ~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\views.py", line 480, in raise_uncaught_exception
    raise exc
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\mixins.py", line 20, in create
    headers = self.get_success_headers(serializer.data)
                                       ^^^^^^^^^^^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\serializers.py", line 555, in data
    ret = super().data
          ^^^^^^^^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\serializers.py", line 253, in data
    self._data = self.to_representation(self.instance)
                 ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\serializers.py", line 509, in to_representation
    attribute = field.get_attribute(instance)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\fields.py", line 446, in get_attribute
    return get_attribute(instance, self.source_attrs)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\fields.py", line 96, in get_attribute
    instance = getattr(instance, attr)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\api\models.py", line 388, in formatted_time
    start_str = self.start_time.strftime('%-I:%M %p')
ValueError: Invalid format string
ERROR 2025-09-26 13:29:02,720 basehttp 20516 23160 "POST /api/events/ HTTP/1.1" 500 149547
INFO 2025-09-26 13:29:11,026 basehttp 20516 23160 "GET /api/contact/ HTTP/1.1" 200 52
INFO 2025-09-26 13:29:11,039 basehttp 20516 23160 "GET /api/contact/ HTTP/1.1" 200 52
ERROR 2025-09-26 13:29:12,093 log 20516 23160 Internal Server Error: /api/events/
Traceback (most recent call last):
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\django\views\decorators\csrf.py", line 65, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\viewsets.py", line 125, in view
    return self.dispatch(request, *args, **kwargs)
           ~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\views.py", line 480, in raise_uncaught_exception
    raise exc
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\api\views.py", line 438, in list
    return Response(serializer.data)
                    ^^^^^^^^^^^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\serializers.py", line 768, in data
    ret = super().data
          ^^^^^^^^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\serializers.py", line 253, in data
    self._data = self.to_representation(self.instance)
                 ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\serializers.py", line 687, in to_representation
    self.child.to_representation(item) for item in iterable
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\serializers.py", line 509, in to_representation
    attribute = field.get_attribute(instance)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\fields.py", line 446, in get_attribute
    return get_attribute(instance, self.source_attrs)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\fields.py", line 96, in get_attribute
    instance = getattr(instance, attr)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\api\models.py", line 388, in formatted_time
    start_str = self.start_time.strftime('%-I:%M %p')
ValueError: Invalid format string
ERROR 2025-09-26 13:29:12,123 basehttp 20516 23160 "GET /api/events/ HTTP/1.1" 500 157003
ERROR 2025-09-26 13:29:12,239 log 20516 23160 Internal Server Error: /api/events/
Traceback (most recent call last):
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\django\views\decorators\csrf.py", line 65, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\viewsets.py", line 125, in view
    return self.dispatch(request, *args, **kwargs)
           ~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\views.py", line 480, in raise_uncaught_exception
    raise exc
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\api\views.py", line 438, in list
    return Response(serializer.data)
                    ^^^^^^^^^^^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\serializers.py", line 768, in data
    ret = super().data
          ^^^^^^^^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\serializers.py", line 253, in data
    self._data = self.to_representation(self.instance)
                 ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\serializers.py", line 687, in to_representation
    self.child.to_representation(item) for item in iterable
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\serializers.py", line 509, in to_representation
    attribute = field.get_attribute(instance)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\fields.py", line 446, in get_attribute
    return get_attribute(instance, self.source_attrs)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\fields.py", line 96, in get_attribute
    instance = getattr(instance, attr)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\api\models.py", line 388, in formatted_time
    start_str = self.start_time.strftime('%-I:%M %p')
ValueError: Invalid format string
ERROR 2025-09-26 13:29:12,249 basehttp 20516 23160 "GET /api/events/ HTTP/1.1" 500 157003
INFO 2025-09-26 13:29:13,025 basehttp 20516 23160 "OPTIONS /api/reservations/ HTTP/1.1" 200 0
INFO 2025-09-26 13:29:13,048 basehttp 20516 23160 "GET /api/reservations/ HTTP/1.1" 200 52
INFO 2025-09-26 13:29:13,064 basehttp 20516 23160 "GET /api/reservations/ HTTP/1.1" 200 52
ERROR 2025-09-26 13:29:13,710 log 20516 16560 Internal Server Error: /api/events/
Traceback (most recent call last):
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\django\views\decorators\csrf.py", line 65, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\viewsets.py", line 125, in view
    return self.dispatch(request, *args, **kwargs)
           ~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\views.py", line 480, in raise_uncaught_exception
    raise exc
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\api\views.py", line 438, in list
    return Response(serializer.data)
                    ^^^^^^^^^^^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\serializers.py", line 768, in data
    ret = super().data
          ^^^^^^^^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\serializers.py", line 253, in data
    self._data = self.to_representation(self.instance)
                 ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\serializers.py", line 687, in to_representation
    self.child.to_representation(item) for item in iterable
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\serializers.py", line 509, in to_representation
    attribute = field.get_attribute(instance)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\fields.py", line 446, in get_attribute
    return get_attribute(instance, self.source_attrs)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\fields.py", line 96, in get_attribute
    instance = getattr(instance, attr)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\api\models.py", line 388, in formatted_time
    start_str = self.start_time.strftime('%-I:%M %p')
ValueError: Invalid format string
ERROR 2025-09-26 13:29:13,715 log 20516 23160 Internal Server Error: /api/events/
Traceback (most recent call last):
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\django\views\decorators\csrf.py", line 65, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\viewsets.py", line 125, in view
    return self.dispatch(request, *args, **kwargs)
           ~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\views.py", line 480, in raise_uncaught_exception
    raise exc
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\api\views.py", line 438, in list
    return Response(serializer.data)
                    ^^^^^^^^^^^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\serializers.py", line 768, in data
    ret = super().data
          ^^^^^^^^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\serializers.py", line 253, in data
    self._data = self.to_representation(self.instance)
                 ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\serializers.py", line 687, in to_representation
    self.child.to_representation(item) for item in iterable
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\serializers.py", line 509, in to_representation
    attribute = field.get_attribute(instance)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\fields.py", line 446, in get_attribute
    return get_attribute(instance, self.source_attrs)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\fields.py", line 96, in get_attribute
    instance = getattr(instance, attr)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\api\models.py", line 388, in formatted_time
    start_str = self.start_time.strftime('%-I:%M %p')
ValueError: Invalid format string
ERROR 2025-09-26 13:29:13,729 basehttp 20516 16560 "GET /api/events/ HTTP/1.1" 500 157003
ERROR 2025-09-26 13:29:13,781 basehttp 20516 23160 "GET /api/events/ HTTP/1.1" 500 157003
WARNING 2025-09-26 13:29:14,315 log 20516 23160 Not Found: /api/reservations/dashboard/
WARNING 2025-09-26 13:29:14,316 log 20516 16560 Not Found: /api/reservations/dashboard/
WARNING 2025-09-26 13:29:14,317 basehttp 20516 23160 "GET /api/reservations/dashboard/ HTTP/1.1" 404 23
WARNING 2025-09-26 13:29:14,352 basehttp 20516 16560 "GET /api/reservations/dashboard/ HTTP/1.1" 404 23
INFO 2025-09-26 13:29:14,799 basehttp 20516 23160 "GET /api/reservations/ HTTP/1.1" 200 52
INFO 2025-09-26 13:29:14,815 basehttp 20516 23160 "GET /api/reservations/ HTTP/1.1" 200 52
ERROR 2025-09-26 13:29:15,283 log 20516 16560 Internal Server Error: /api/events/
Traceback (most recent call last):
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\django\views\decorators\csrf.py", line 65, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\viewsets.py", line 125, in view
    return self.dispatch(request, *args, **kwargs)
           ~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\views.py", line 480, in raise_uncaught_exception
    raise exc
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\api\views.py", line 438, in list
    return Response(serializer.data)
                    ^^^^^^^^^^^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\serializers.py", line 768, in data
    ret = super().data
          ^^^^^^^^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\serializers.py", line 253, in data
    self._data = self.to_representation(self.instance)
                 ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\serializers.py", line 687, in to_representation
    self.child.to_representation(item) for item in iterable
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\serializers.py", line 509, in to_representation
    attribute = field.get_attribute(instance)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\fields.py", line 446, in get_attribute
    return get_attribute(instance, self.source_attrs)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\fields.py", line 96, in get_attribute
    instance = getattr(instance, attr)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\api\models.py", line 388, in formatted_time
    start_str = self.start_time.strftime('%-I:%M %p')
ValueError: Invalid format string
ERROR 2025-09-26 13:29:15,312 log 20516 23160 Internal Server Error: /api/events/
Traceback (most recent call last):
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\django\views\decorators\csrf.py", line 65, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\viewsets.py", line 125, in view
    return self.dispatch(request, *args, **kwargs)
           ~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\views.py", line 480, in raise_uncaught_exception
    raise exc
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\api\views.py", line 438, in list
    return Response(serializer.data)
                    ^^^^^^^^^^^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\serializers.py", line 768, in data
    ret = super().data
          ^^^^^^^^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\serializers.py", line 253, in data
    self._data = self.to_representation(self.instance)
                 ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\serializers.py", line 687, in to_representation
    self.child.to_representation(item) for item in iterable
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\serializers.py", line 509, in to_representation
    attribute = field.get_attribute(instance)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\fields.py", line 446, in get_attribute
    return get_attribute(instance, self.source_attrs)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\fields.py", line 96, in get_attribute
    instance = getattr(instance, attr)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\api\models.py", line 388, in formatted_time
    start_str = self.start_time.strftime('%-I:%M %p')
ValueError: Invalid format string
ERROR 2025-09-26 13:29:15,323 basehttp 20516 16560 "GET /api/events/ HTTP/1.1" 500 157003
ERROR 2025-09-26 13:29:15,352 basehttp 20516 23160 "GET /api/events/ HTTP/1.1" 500 157003
INFO 2025-09-26 13:29:16,774 basehttp 20516 23160 "GET /api/contact/ HTTP/1.1" 200 52
INFO 2025-09-26 13:29:16,797 basehttp 20516 23160 "GET /api/contact/ HTTP/1.1" 200 52
INFO 2025-09-26 13:29:17,421 basehttp 20516 23160 "OPTIONS /api/dashboard/analytics/ HTTP/1.1" 200 0
INFO 2025-09-26 13:29:17,502 basehttp 20516 23160 "GET /api/dashboard/analytics/ HTTP/1.1" 200 192
INFO 2025-09-26 13:29:17,563 basehttp 20516 23160 "GET /api/dashboard/analytics/ HTTP/1.1" 200 192
INFO 2025-09-26 13:29:19,159 basehttp 20516 23160 "GET /api/dashboard/analytics/ HTTP/1.1" 200 192
INFO 2025-09-26 13:29:19,220 basehttp 20516 23160 "GET /api/dashboard/analytics/ HTTP/1.1" 200 192
WARNING 2025-09-26 13:29:26,661 log 20516 23160 Not Found: /api/reservations/dashboard/
WARNING 2025-09-26 13:29:26,663 log 20516 16560 Not Found: /api/reservations/dashboard/
WARNING 2025-09-26 13:29:26,665 basehttp 20516 23160 "GET /api/reservations/dashboard/ HTTP/1.1" 404 23
WARNING 2025-09-26 13:29:26,673 basehttp 20516 16560 "GET /api/reservations/dashboard/ HTTP/1.1" 404 23
INFO 2025-09-26 13:29:28,612 basehttp 20516 23160 "GET /api/contact/ HTTP/1.1" 200 52
INFO 2025-09-26 13:29:28,630 basehttp 20516 23160 "GET /api/contact/ HTTP/1.1" 200 52
ERROR 2025-09-26 13:29:30,396 log 20516 23160 Internal Server Error: /api/events/
Traceback (most recent call last):
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\django\views\decorators\csrf.py", line 65, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\viewsets.py", line 125, in view
    return self.dispatch(request, *args, **kwargs)
           ~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\views.py", line 480, in raise_uncaught_exception
    raise exc
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\api\views.py", line 438, in list
    return Response(serializer.data)
                    ^^^^^^^^^^^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\serializers.py", line 768, in data
    ret = super().data
          ^^^^^^^^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\serializers.py", line 253, in data
    self._data = self.to_representation(self.instance)
                 ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\serializers.py", line 687, in to_representation
    self.child.to_representation(item) for item in iterable
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\serializers.py", line 509, in to_representation
    attribute = field.get_attribute(instance)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\fields.py", line 446, in get_attribute
    return get_attribute(instance, self.source_attrs)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\fields.py", line 96, in get_attribute
    instance = getattr(instance, attr)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\api\models.py", line 388, in formatted_time
    start_str = self.start_time.strftime('%-I:%M %p')
ValueError: Invalid format string
ERROR 2025-09-26 13:29:30,419 log 20516 16560 Internal Server Error: /api/events/
Traceback (most recent call last):
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\django\views\decorators\csrf.py", line 65, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\viewsets.py", line 125, in view
    return self.dispatch(request, *args, **kwargs)
           ~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\views.py", line 480, in raise_uncaught_exception
    raise exc
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\api\views.py", line 438, in list
    return Response(serializer.data)
                    ^^^^^^^^^^^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\serializers.py", line 768, in data
    ret = super().data
          ^^^^^^^^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\serializers.py", line 253, in data
    self._data = self.to_representation(self.instance)
                 ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\serializers.py", line 687, in to_representation
    self.child.to_representation(item) for item in iterable
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\serializers.py", line 509, in to_representation
    attribute = field.get_attribute(instance)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\fields.py", line 446, in get_attribute
    return get_attribute(instance, self.source_attrs)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\fields.py", line 96, in get_attribute
    instance = getattr(instance, attr)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\api\models.py", line 388, in formatted_time
    start_str = self.start_time.strftime('%-I:%M %p')
ValueError: Invalid format string
ERROR 2025-09-26 13:29:30,440 basehttp 20516 23160 "GET /api/events/ HTTP/1.1" 500 157003
ERROR 2025-09-26 13:29:30,598 basehttp 20516 16560 "GET /api/events/ HTTP/1.1" 500 157003
ERROR 2025-09-26 13:29:46,042 log 20516 23160 Internal Server Error: /api/events/
Traceback (most recent call last):
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\django\views\decorators\csrf.py", line 65, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\viewsets.py", line 125, in view
    return self.dispatch(request, *args, **kwargs)
           ~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\views.py", line 480, in raise_uncaught_exception
    raise exc
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\mixins.py", line 20, in create
    headers = self.get_success_headers(serializer.data)
                                       ^^^^^^^^^^^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\serializers.py", line 555, in data
    ret = super().data
          ^^^^^^^^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\serializers.py", line 253, in data
    self._data = self.to_representation(self.instance)
                 ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\serializers.py", line 509, in to_representation
    attribute = field.get_attribute(instance)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\fields.py", line 446, in get_attribute
    return get_attribute(instance, self.source_attrs)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\fields.py", line 96, in get_attribute
    instance = getattr(instance, attr)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\api\models.py", line 388, in formatted_time
    start_str = self.start_time.strftime('%-I:%M %p')
ValueError: Invalid format string
ERROR 2025-09-26 13:29:46,048 basehttp 20516 23160 "POST /api/events/ HTTP/1.1" 500 149573
ERROR 2025-09-26 13:29:55,252 log 20516 23160 Internal Server Error: /api/events/public/
Traceback (most recent call last):
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\django\views\decorators\csrf.py", line 65, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\viewsets.py", line 125, in view
    return self.dispatch(request, *args, **kwargs)
           ~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\views.py", line 480, in raise_uncaught_exception
    raise exc
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\api\views.py", line 498, in public
    return Response(serializer.data)
                    ^^^^^^^^^^^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\serializers.py", line 768, in data
    ret = super().data
          ^^^^^^^^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\serializers.py", line 253, in data
    self._data = self.to_representation(self.instance)
                 ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\serializers.py", line 687, in to_representation
    self.child.to_representation(item) for item in iterable
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\serializers.py", line 509, in to_representation
    attribute = field.get_attribute(instance)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\fields.py", line 446, in get_attribute
    return get_attribute(instance, self.source_attrs)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\fields.py", line 96, in get_attribute
    instance = getattr(instance, attr)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\api\models.py", line 388, in formatted_time
    start_str = self.start_time.strftime('%-I:%M %p')
ValueError: Invalid format string
ERROR 2025-09-26 13:29:55,302 basehttp 20516 23160 "GET /api/events/public/ HTTP/1.1" 500 152843
ERROR 2025-09-26 13:29:56,109 log 20516 23160 Internal Server Error: /api/events/public/
Traceback (most recent call last):
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\django\views\decorators\csrf.py", line 65, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\viewsets.py", line 125, in view
    return self.dispatch(request, *args, **kwargs)
           ~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\views.py", line 480, in raise_uncaught_exception
    raise exc
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\api\views.py", line 498, in public
    return Response(serializer.data)
                    ^^^^^^^^^^^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\serializers.py", line 768, in data
    ret = super().data
          ^^^^^^^^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\serializers.py", line 253, in data
    self._data = self.to_representation(self.instance)
                 ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\serializers.py", line 687, in to_representation
    self.child.to_representation(item) for item in iterable
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\serializers.py", line 509, in to_representation
    attribute = field.get_attribute(instance)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\fields.py", line 446, in get_attribute
    return get_attribute(instance, self.source_attrs)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\venv\Lib\site-packages\rest_framework\fields.py", line 96, in get_attribute
    instance = getattr(instance, attr)
  File "C:\Users\Perdoruess\bottlebrothers_lab-7\server\api\models.py", line 388, in formatted_time
    start_str = self.start_time.strftime('%-I:%M %p')
ValueError: Invalid format string
ERROR 2025-09-26 13:29:56,353 basehttp 20516 23160 "GET /api/events/public/ HTTP/1.1" 500 152843
INFO 2025-09-26 13:35:16,365 autoreload 20516 10452 C:\Users\Perdoruess\bottlebrothers_lab-7\server\api\models.py changed, reloading.
INFO 2025-09-26 13:35:21,446 autoreload 27960 3384 Watching for file changes with StatReloader
INFO 2025-09-26 13:35:58,984 autoreload 10636 27940 Watching for file changes with StatReloader
INFO 2025-09-26 13:36:03,416 basehttp 10636 22640 "POST /api/events/ HTTP/1.1" 201 992
INFO 2025-09-26 13:36:03,452 basehttp 10636 22640 "GET /api/events/ HTTP/1.1" 200 15763
INFO 2025-09-26 13:36:08,733 basehttp 10636 22640 "GET /api/events/public/ HTTP/1.1" 200 10771
INFO 2025-09-26 13:36:09,358 basehttp 10636 29348 "GET /media/events/0c9fb13319614d00bf5f74ca0a52f70a.png HTTP/1.1" 200 219903
INFO 2025-09-26 13:36:09,472 basehttp 10636 29348 "GET /media/events/5039bf55d9df4ffa9b80e130cbe3c50d.png HTTP/1.1" 200 219903
INFO 2025-09-26 13:36:09,577 basehttp 10636 29348 "GET /media/events/81e72cf4dd314791b52c5cc59c376484.png HTTP/1.1" 200 219903
INFO 2025-09-26 13:36:09,608 basehttp 10636 17884 "GET /media/events/91cd076db46a4be59cf95a3df15f64de.png HTTP/1.1" 200 219903
INFO 2025-09-26 13:36:09,617 basehttp 10636 29348 "GET /media/events/8f2bf2464da642a99d6abd8a3acc8c47.png HTTP/1.1" 200 219903
INFO 2025-09-26 13:36:09,669 basehttp 10636 17884 "GET /media/events/4edacdffa82344189caef8d21cb6584e.png HTTP/1.1" 200 219903
INFO 2025-09-26 13:36:09,683 basehttp 10636 20528 "GET /media/events/b5fb225f3f0a4cbeb4fa8acdf842fe30.png HTTP/1.1" 200 219903
INFO 2025-09-26 13:36:09,726 basehttp 10636 24884 "GET /media/events/4d81e363c0d74f39875aa151d877ace9.png HTTP/1.1" 200 219903
INFO 2025-09-26 13:36:09,825 basehttp 10636 29348 "GET /media/events/c92b40fc0e61457e8f4b6814e0fc062d.png HTTP/1.1" 200 219903
INFO 2025-09-26 13:36:09,927 basehttp 10636 17884 "GET /media/events/5c75b82dedcb4d05b08dfa92ea06606e.png HTTP/1.1" 200 219903
INFO 2025-09-26 13:36:09,953 basehttp 10636 25488 "GET /media/events/76f39afdfd814289b767ca4f7570bfa9.png HTTP/1.1" 200 219903
INFO 2025-09-26 13:36:09,958 basehttp 10636 20528 "GET /media/events/91a7232afa4a45688e30b546adf8fa0f.png HTTP/1.1" 200 219903
INFO 2025-09-26 13:36:10,009 basehttp 10636 24884 "GET /media/events/656c68cf2b2948858f7bcc32ae8ff453.png HTTP/1.1" 200 219903
INFO 2025-09-26 13:36:10,020 basehttp 10636 29348 "GET /media/events/4fad93826e4e40d58d55f4eee5a07496.png HTTP/1.1" 200 219903
INFO 2025-09-26 13:36:10,067 basehttp 10636 17884 "GET /media/events/0a4b3d13a781410c962202c8a6a9d07c.png HTTP/1.1" 200 219903
INFO 2025-09-26 13:36:10,289 basehttp 10636 20532 "GET /media/events/f916337792ef47f798db8a694cafe702.png HTTP/1.1" 200 219903
INFO 2025-09-26 13:37:04,030 basehttp 10636 22640 "POST /api/events/ HTTP/1.1" 201 1030
INFO 2025-09-26 13:37:04,075 basehttp 10636 22640 "GET /api/events/ HTTP/1.1" 200 16794
INFO 2025-09-26 13:37:19,809 basehttp 10636 22640 "OPTIONS /api/events/0aefdc65-8d77-40d6-980a-79a8f0e6edb1/ HTTP/1.1" 200 0
INFO 2025-09-26 13:37:19,837 basehttp 10636 22640 "DELETE /api/events/0aefdc65-8d77-40d6-980a-79a8f0e6edb1/ HTTP/1.1" 204 0
INFO 2025-09-26 13:37:19,877 basehttp 10636 22640 "GET /api/events/ HTTP/1.1" 200 15810
INFO 2025-09-26 13:37:22,775 basehttp 10636 22640 "OPTIONS /api/events/f3737852-6749-41f1-ac0f-26904edd8af4/ HTTP/1.1" 200 0
INFO 2025-09-26 13:37:22,810 basehttp 10636 22640 "DELETE /api/events/f3737852-6749-41f1-ac0f-26904edd8af4/ HTTP/1.1" 204 0
INFO 2025-09-26 13:37:22,851 basehttp 10636 22640 "GET /api/events/ HTTP/1.1" 200 14826
INFO 2025-09-26 13:37:25,020 basehttp 10636 22640 "OPTIONS /api/events/90f6d8c0-3936-4f18-9881-8eb738c305f0/ HTTP/1.1" 200 0
INFO 2025-09-26 13:37:25,043 basehttp 10636 22640 "DELETE /api/events/90f6d8c0-3936-4f18-9881-8eb738c305f0/ HTTP/1.1" 204 0
INFO 2025-09-26 13:37:25,116 basehttp 10636 22640 "GET /api/events/ HTTP/1.1" 200 13842
INFO 2025-09-26 13:37:27,416 basehttp 10636 22640 "OPTIONS /api/events/d665f943-1a8d-4c64-a9d4-ab25abed661b/ HTTP/1.1" 200 0
INFO 2025-09-26 13:37:27,447 basehttp 10636 22640 "DELETE /api/events/d665f943-1a8d-4c64-a9d4-ab25abed661b/ HTTP/1.1" 204 0
INFO 2025-09-26 13:37:27,515 basehttp 10636 22640 "GET /api/events/ HTTP/1.1" 200 12858
INFO 2025-09-26 13:37:30,189 basehttp 10636 22640 "OPTIONS /api/events/c68f4395-95f6-4f29-8968-2deee9f5eac3/ HTTP/1.1" 200 0
INFO 2025-09-26 13:37:30,221 basehttp 10636 22640 "DELETE /api/events/c68f4395-95f6-4f29-8968-2deee9f5eac3/ HTTP/1.1" 204 0
INFO 2025-09-26 13:37:30,291 basehttp 10636 22640 "GET /api/events/ HTTP/1.1" 200 11874
INFO 2025-09-26 13:37:31,955 basehttp 10636 22640 "OPTIONS /api/events/2d1258d4-b192-4941-9704-bcd204ebd1d1/ HTTP/1.1" 200 0
INFO 2025-09-26 13:37:31,978 basehttp 10636 22640 "DELETE /api/events/2d1258d4-b192-4941-9704-bcd204ebd1d1/ HTTP/1.1" 204 0
INFO 2025-09-26 13:37:32,035 basehttp 10636 22640 "GET /api/events/ HTTP/1.1" 200 10890
INFO 2025-09-26 13:37:34,027 basehttp 10636 22640 "OPTIONS /api/events/57bd4384-a98b-4282-ab0f-f6c0388b16a0/ HTTP/1.1" 200 0
INFO 2025-09-26 13:37:34,050 basehttp 10636 22640 "DELETE /api/events/57bd4384-a98b-4282-ab0f-f6c0388b16a0/ HTTP/1.1" 204 0
INFO 2025-09-26 13:37:34,136 basehttp 10636 22640 "GET /api/events/ HTTP/1.1" 200 9906
INFO 2025-09-26 13:37:35,930 basehttp 10636 22640 "OPTIONS /api/events/a4cde419-a363-4984-86f1-024086e2199d/ HTTP/1.1" 200 0
INFO 2025-09-26 13:37:35,961 basehttp 10636 22640 "DELETE /api/events/a4cde419-a363-4984-86f1-024086e2199d/ HTTP/1.1" 204 0
INFO 2025-09-26 13:37:36,010 basehttp 10636 22640 "GET /api/events/ HTTP/1.1" 200 8922
INFO 2025-09-26 13:37:38,077 basehttp 10636 22640 "OPTIONS /api/events/67726ee9-33f9-4d59-9549-f033fb5ea856/ HTTP/1.1" 200 0
INFO 2025-09-26 13:37:38,104 basehttp 10636 22640 "DELETE /api/events/67726ee9-33f9-4d59-9549-f033fb5ea856/ HTTP/1.1" 204 0
INFO 2025-09-26 13:37:38,155 basehttp 10636 22640 "GET /api/events/ HTTP/1.1" 200 7938
INFO 2025-09-26 13:37:41,513 basehttp 10636 22640 "GET /api/events/public/ HTTP/1.1" 200 5442
INFO 2025-09-26 13:37:41,683 basehttp 10636 17736 "GET /api/events/public/ HTTP/1.1" 200 5442
INFO 2025-09-26 13:37:41,785 basehttp 10636 25488 "GET /media/events/4edacdffa82344189caef8d21cb6584e.png HTTP/1.1" 304 0
INFO 2025-09-26 13:37:41,801 basehttp 10636 20528 "GET /media/events/c92b40fc0e61457e8f4b6814e0fc062d.png HTTP/1.1" 304 0
INFO 2025-09-26 13:37:41,812 basehttp 10636 24884 "GET /media/events/5c75b82dedcb4d05b08dfa92ea06606e.png HTTP/1.1" 304 0
INFO 2025-09-26 13:37:41,832 basehttp 10636 29348 "GET /media/events/91a7232afa4a45688e30b546adf8fa0f.png HTTP/1.1" 304 0
INFO 2025-09-26 13:37:41,878 basehttp 10636 17884 "GET /media/events/656c68cf2b2948858f7bcc32ae8ff453.png HTTP/1.1" 304 0
INFO 2025-09-26 13:37:41,935 basehttp 10636 20532 "GET /media/events/4fad93826e4e40d58d55f4eee5a07496.png HTTP/1.1" 304 0
INFO 2025-09-26 13:37:41,941 basehttp 10636 25488 "GET /media/events/0a4b3d13a781410c962202c8a6a9d07c.png HTTP/1.1" 304 0
INFO 2025-09-26 13:37:42,103 basehttp 10636 20528 "GET /media/events/afb48390c3a0430aafeb866a186248b9.png HTTP/1.1" 200 105790
INFO 2025-09-26 13:38:18,579 basehttp 10636 17736 "GET /api/events/public/ HTTP/1.1" 200 5442
INFO 2025-09-26 13:38:18,609 basehttp 10636 17736 "GET /api/events/public/ HTTP/1.1" 200 5442
INFO 2025-09-26 13:38:19,351 basehttp 10636 17736 "GET /api/gallery/public/ HTTP/1.1" 200 458
INFO 2025-09-26 13:38:19,367 basehttp 10636 17736 "GET /api/gallery/public/ HTTP/1.1" 200 458
WARNING 2025-09-26 13:38:19,564 log 10636 24884 Not Found: /media/gallery/4977b4d89700444f99604571036d7f33.png
WARNING 2025-09-26 13:38:19,567 basehttp 10636 24884 "GET /media/gallery/4977b4d89700444f99604571036d7f33.png HTTP/1.1" 404 2805
INFO 2025-09-26 13:38:20,427 basehttp 10636 17736 "GET /api/events/public/ HTTP/1.1" 200 5442
INFO 2025-09-26 13:38:20,449 basehttp 10636 17736 "GET /api/events/public/ HTTP/1.1" 200 5442
INFO 2025-09-26 13:38:21,082 basehttp 10636 17736 "GET /api/gallery/public/ HTTP/1.1" 200 458
INFO 2025-09-26 13:38:21,118 basehttp 10636 17736 "GET /api/gallery/public/ HTTP/1.1" 200 458
WARNING 2025-09-26 13:38:21,352 log 10636 29348 Not Found: /media/gallery/4977b4d89700444f99604571036d7f33.png
WARNING 2025-09-26 13:38:21,360 basehttp 10636 29348 "GET /media/gallery/4977b4d89700444f99604571036d7f33.png HTTP/1.1" 404 2805
WARNING 2025-09-26 13:38:25,824 log 10636 17884 Not Found: /media/gallery/4977b4d89700444f99604571036d7f33.png
WARNING 2025-09-26 13:38:25,835 basehttp 10636 17884 "GET /media/gallery/4977b4d89700444f99604571036d7f33.png HTTP/1.1" 404 2805
INFO 2025-09-26 13:38:35,996 basehttp 10636 17736 "GET /api/events/public/ HTTP/1.1" 200 5442
INFO 2025-09-26 13:38:36,024 basehttp 10636 17736 "GET /api/events/public/ HTTP/1.1" 200 5442
